hack/local-setup.sh

```diff
@@ -254,7 +256,7 @@ deployOCMSpoke(){
   kubectl config use-context kind-${KIND_CLUSTER_CONTROL_PLANE}
   join=$(${CLUSTERADM_BIN} get token --context kind-${KIND_CLUSTER_CONTROL_PLANE} |  grep -o  'clusteradm.*--cluster-name')
   kubectl config use-context kind-${clusterName}
-  ${BIN_DIR}/${join} kind-${clusterName} --force-internal-endpoint-lookup --context kind-${clusterName} | grep clusteradm
+  ${BIN_DIR}/${join} kind-${clusterName} --bundle-version='0.11.0' --feature-gates=RawFeedbackJsonString=true --force-internal-endpoint-lookup --context kind-${clusterName} | grep clusteradm
```

Comment:
wondering if these flags can be set for existing clusters? If not we might need to be able to fall back to the old way

Reply:
The Klusterlet resource can be modified.
Here's an example of one where the gate is enabled

```yaml
apiVersion: operator.open-cluster-management.io/v1
kind: Klusterlet
metadata:
  name: klusterlet
spec:
  clusterName: kind-mgc-control-plane
  namespace: open-cluster-management-agent
  registrationConfiguration:
    featureGates:
    - feature: AddonManagement
      mode: Enable
  workConfiguration:
    featureGates:
    - feature: RawFeedbackJsonString
      mode: Enable
```